crates/vm/levm/src/opcode_handlers/stack_memory_storage_flow.rs

```diff
@@ -329,11 +330,45 @@ impl VM {
         self.increase_consumed_gas(current_call_frame, gas_cost::JUMP)?;
 
         let jump_address = current_call_frame.stack.pop()?;
-        current_call_frame.jump(jump_address)?;
+        Self::jump(current_call_frame, jump_address)?;
 
         Ok(OpcodeSuccess::Continue)
     }
 
+    /// JUMP* family (`JUMP` and `JUMP` ATTOW [DEC 2024]) helper
+    /// function.
+    /// This function returns whether the `address` is a valid JUMPDEST
+    /// for the specified `stack_frame` or not.
+    fn is_valid_jump_addr(stack_frame: &CallFrame, address: usize) -> bool {
+        matches!(
+            stack_frame
+                .bytecode
+                .get(address)
+                .copied()
+                .map(Opcode::try_from),
+            Some(Ok(Opcode::JUMPDEST))
+        )
+    }
+
+    /// JUMP* family (`JUMP` and `JUMP` ATTOW [DEC 2024]) helper
+    /// function.
+    /// This function will change the PC for the specified stack frame
+    /// to be equal to the specified address. If the address is a
+    /// valid JUMPDEST, it will return an error
```

Comment:
```suggestion
    /// to be equal to the specified address. If the address is not a
    /// valid JUMPDEST, it will return an error
```

Reply:
Yikes! What a catch Jere! Fixed in latest commit